maintenance_plan_meter_mode/views/maintenance_views.xml

```diff
@@ -81,9 +85,28 @@
         <field name="arch" type="xml">
             <xpath expr="//field[@name='planning_step']" position="after">
                 <field name="meter_mode"/>
-                <field name="meter_current_value"/>
-                <field name="meter_autoinc_maxvalue"/>
-                <field name="meter_autoinc_lastvalue"/>
+            </xpath>
+            <xpath expr="//notebook" position="inside">
+                <page string="Maintenance Meter Mode">
+                    <group name="maintenance_meter_mode">
+                        <field name="maintenance_meter_mode_plan_ids" nolabel="1"
+                               context="{'default_equipment_id': active_id, 'hide_equipment_id': 1, 'default_meter_mode': True}">
+                            <tree string="Meter mode Plans">
+                                <field name="maintenance_kind_id"/>
+                                <field name="name"/>
+                                <field name="meter_mode"/>
```

Comment:
This field could be removed from the view, only meter mode plans are actually shown